test/endToEnd/authorization/organizationsTest.js

```diff
@@ -0,0 +1,289 @@
+const Lab = require('lab')
+const lab = exports.lab = Lab.script()
+
+const orgOps = require('../../../src/lib/ops/organizationOps')
+const server = require('./../../../src/wiring-hapi')
+const Factory = require('../../factory')
+const BuildFor = require('./testBuilder')
+
+const organizationId = 'WONKA'
+function Policy (Statement) {
+  return {
+    version: '2016-07-01',
+    name: 'Test Policy',
+    statements: JSON.stringify({
+      Statement: Statement || [{
+        Effect: 'Allow',
+        Action: ['dummy'],
+        Resource: ['dummy']
+      }]
+    }),
+    organizationId
+  }
+}
+
+lab.experiment('Routes Authorizations', () => {
+  lab.experiment('organizations', () => {
+    lab.experiment('GET ', () => {
+
+      const records = Factory(lab, {
+        users: {
+          caller: { name: 'caller', organizationId, policies: ['testedPolicy'] }
+        },
+        policies: {
+          testedPolicy: Policy()
+        }
+      })
+
+      const endpoint = BuildFor(lab, records)
+        .server(server)
+        .endpoint({
+          method: 'GET',
+          url: '/authorization/organizations',
+          headers: { authorization: '{{caller.id}}' }
+        })
+
+      endpoint.test('should authorize user with correct policy')
+        .withPolicy([{
+          Effect: 'Allow',
+          Action: ['authorization:organizations:list'],
+          Resource: ['*']
```

Comment:
Yes, you're right. I was fooled by the fact that the route definition doesn't specify a resource and I forgot the auto-build mechanism. Will fix